speculation-rules/prerender/service-workers.https.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<title>Same-origin prerendering can access Indexed Database</title>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="resources/utils.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<body>
+<script>
+
+setup(() => assertSpeculationRulesIsSupported());
+
+const ACTIVATION_DELAY = 500;
```

Comment:
Can we expressly wait for the event we want? If we can't now, can we explain why and consider a TODO for doing something cleaner? Or is there a reason for this constant delay?

Reply:
Added a comment.  I can't think of another way to test an expected deference, except for waiting and failing if the deferred operation occurred before its precondition.